apache2/msc_crypt.c

```diff
@@ -1252,6 +1252,17 @@ char *do_hash_link(modsec_rec *msr, char *link, int type)  {
 
     if(msr == NULL) return NULL;
 
+    /* crypto_key optimization */
```

Comment:
If this is a permanent change, then this comment is not very meaningful. I suggest to remove it.

Reply:
will remove it